src/Avalonia.Diagnostics/Diagnostics/Views/ControlDetailsView.xaml

```diff
@@ -4,15 +4,21 @@
              xmlns:local="clr-namespace:Avalonia.Diagnostics.Views"
              xmlns:controls="clr-namespace:Avalonia.Diagnostics.Controls"
              xmlns:vm="clr-namespace:Avalonia.Diagnostics.ViewModels"
+             xmlns:lb="using:Avalonia.Diagnostics.Behaviors"
              x:Class="Avalonia.Diagnostics.Views.ControlDetailsView"
              x:Name="Main">
 
   <UserControl.Resources>
     <conv:BoolToOpacityConverter x:Key="BoolToOpacity" Opacity="0.6"/>
   </UserControl.Resources>
 
-  <Grid ColumnDefinitions="*,Auto,320">
-
+  <Grid>
+    <Grid.ColumnDefinitions>
+      <ColumnDefinition Width="*"/>
+      <ColumnDefinition Width="Auto"/>
+      <ColumnDefinition Width="320" lb:ColumnDefinition.IsVisible="{Binding Layout, Converter={x:Static ObjectConverters.IsNotNull}}" />
```

Comment:
Can you add a comment here, why we need ColumnDefinion.IsVisible? And probably to that class as well.
Is it because we can't set default Width="320" (while column will be Auto, so with hidden grid it will collapse as well) on the grid itself and still be able to use GridSplitter?